pkg/monitor/monitor/util.go

```diff
@@ -726,9 +726,20 @@ func getMonitorGrafanaContainer(secret *core.Secret, monitor *v1alpha1.TidbMonit
 }
 
 func getMonitorPrometheusReloaderContainer(monitor *v1alpha1.TidbMonitor, shard int32) core.Container {
+	image := "quay.io/prometheus-operator/prometheus-config-reloader"
```

Comment:
Some users cannot access this image, so after upgrading the TiDB Operator, their tidb monitors will fail to start, I think it's OK that we add the reloader configuration in the examples but not enforce this in the code.

Reply:
ok，you‘re right.